Cesium3DTiles/src/Tileset.cpp

```diff
@@ -1224,18 +1236,30 @@ Tileset::TraversalDetails Tileset::_visitTileIfNeeded(
   tile.update(frameState.lastFrameNumber, frameState.currentFrameNumber);
   this->_markTileVisited(tile);
 
+  const Tileset* pTileset = tile.getTileset();
+  if (!pTileset) {
+    return TraversalDetails();
+  }
+
   // whether we should visit this tile
   bool shouldVisit = true;
   // whether this tile was culled (Note: we might still want to visit it)
   bool culled = false;
 
   const BoundingVolume& boundingVolume = tile.getBoundingVolume();
-  const ViewState& viewState = frameState.viewState;
-
-  if (!isVisibleFromCamera(
-          frameState.viewState,
-          boundingVolume,
-          this->_options.renderTilesUnderCamera)) {
+  const std::vector<ViewState>& frustums = frameState.frustums;
+
+  if (std::none_of(
+          frustums.begin(),
+          frustums.end(),
+          [boundingVolume,
+           renderTilesUnderCamera = this->_options.renderTilesUnderCamera](
+              const ViewState& frustum) {
```

Comment:
fwiw I quite like this functional-ish style.

Reply:
I don't have strong opinion on this, so if everyone is good with it, then it's okay for me

Reply:
Subjetively, I also like this functional style (and it has several advantages - there's a compelling reason for nearly all major programming languages having moved into that direction since 2005 - "The free lunch is over", roughly related to "internal iteration" vs "external iteration" and such...)

**But** ... I don't like large, anonymous lambdas. This specific block would IMHO be *much* better if it was just a line 
```
if (std::none_of(frustums.begin(), frustums.end(), tileIsInFrustum)) {
    ...
}
```
with a proper `auto tileIsInFrustum = ...`  lambda.